index.js

```diff
@@ -1,32 +1 @@
-/**
- * @typedef {Pick<
- *   import('unified-engine').Options,
- *   'ignoreName' | 'packageField' | 'pluginPrefix' | 'plugins' | 'rcName'
- * > & {
- *   defaultSource?: string
- * }} Options
- */
-
-import {
-  createConnection,
-  ProposedFeatures,
-  TextDocuments
-} from 'vscode-languageserver/node.js'
-import {TextDocument} from 'vscode-languageserver-textdocument'
-import {configureUnifiedLanguageServer} from './lib/server.js'
-
-/**
- * Create a language server for a unified ecosystem.
- *
- * @param {Options} options
- *   Configuration for `unified-engine` and the language server.
- */
-export function createUnifiedLanguageServer(options) {
-  const connection = createConnection(ProposedFeatures.all)
-  const documents = new TextDocuments(TextDocument)
-
-  configureUnifiedLanguageServer(connection, documents, options)
-
-  documents.listen(connection)
-  connection.listen()
-}
+export {configureUnifiedLanguageServer} from './lib/index.js'
```

Comment:
```suggestion
/**
 * @typedef {import('./lib/index.js').Options} Options
 */

export {configureUnifiedLanguageServer} from './lib/index.js'
```

The `Options` type is still useful though?
I’m sorry if what I said earlier was confusing, I meant to suggest to define types where they are in code, while still exporting them to API users